vnavmesh/VNavmeshIPC.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Navmesh.Movement;
+using System.Numerics;
+
+namespace Navmesh
+{
+    internal static class VNavmeshIPC
```

Comment:
Does it have to be static, rather than a private instance owned by Plugin? I find it weird to have a static Dispose(), it doesn't seem very conventional.

Reply:
I just made it static because none of the methods/fields are referencing any instance methods or fields and this class only needs to have the single instance, I'm not sure even having the ability to instance this class would be a good idea because I don't think dalamud would like multiple subscriptions (just an assumption have not tried lmao), also the dispose was just named that for convenience it doesn't actually implement IDisposible and is just to unsubscribe the ipc's it could be renamed. That being said it's your baby if you would like me to change it to instanced, I have no issue with that.